### ISSUE TITLE
Custom field, fields autoloader, file naming.

### ISSUE THREAD

**Issue by slawekniewiadomski** [siteorigin/docs#39]
Guys, i almost got bald because of this today:
When adding custom field, file containing field class, must have `.class.php ` extension!
As stands in **siteorigin-widget-field-class-loader.class.php:**
```php 
//(...)
$filepath = $class_path . $filename . '.class.php';
//(...)
```
**IMO this should be mentioned in docs, bold!**, or fields autoloader should look for files `field-name.class.php` OR ` field-name.php ` 

KUDOS to this guy, without his post i would never suspect bad filename.: https://siteorigin.com/thread/custom-field-i-created-always-returns-an-error-in-the-backend/

**Comment by Misplon** [comment]
@slawekniewiadomski thanks for your feedback. Sorry you had to go through this. We'll improve the docs ASAP.

**Comment by slawekniewiadomski** [comment]
@Misplon thx for answer, and thx for wonderful piece of code!
If you dont have any big plans about docs, after the weekend (~tuesday) i shall send appropriate pull request.

**Comment by Misplon** [comment]
Thanks for your understanding. I've chatted to the team and we realise we're behind. There is a big rewrite on the way soon but I don't have an exact date. We're a very small team at the moment. If you had the time and energy to send a PR it would be most appreciated.

**Comment by AlexGStapleton** [comment]
Resolved in #40
